### ISSUE TITLE
Stored Object: paper:arxiv.2604.08423

### ISSUE THREAD

**Issue by chtmp223** [chtmp223/papers-feed#296]
{
  "sourceId": "arxiv",
  "paperId": "2604.08423",
  "url": "https://arxiv.org/abs/2604.08423",
  "title": "Synthetic Data for any Differentiable Target",
  "authors": "Tristan Thrush, Sung Min Park, Herman Brunborg, Luke Bailey, Marcel Roed, Neil Band, Christopher Potts, Tatsunori Hashimoto",
  "abstract": "What are the limits of controlling language models via synthetic training data? We develop a reinforcement learning (RL) primitive, the Dataset Policy Gradient (DPG), which can precisely optimize synthetic data generators to produce a dataset of targeted examples. When used for supervised fine-tuning (SFT) of a target model, these examples cause the target model to do well on a differentiable metric of our choice. Our approach achieves this by taking exact data attribution via higher-order gradients and using those scores as policy gradient rewards. We prove that this procedure closely approximates the true, intractable gradient for the synthetic data generator. To illustrate the potential of DPG, we show that, using only SFT on generated examples, we can cause the target model's LM head weights to (1) embed a QR code, (2) embed the pattern \ud835\udffc\ud835\udffd\\texttt{67}, and (3) have lower \u21132\\ell^2 norm. We additionally show that we can cause the generator to (4) rephrase inputs in a new language and (5) produce a specific UUID, even though neither of these objectives is conveyed in the generator's input prompts. These findings suggest that DPG is a powerful and flexible technique for shaping model properties using only synthetic training examples.",
  "timestamp": "2026-04-10T18:06:15.607Z",
  "rating": "novote",
  "publishedDate": "2026/04/09",
  "tags": [
    "Computation and Language (cs.CL)",
    "Artificial Intelligence (cs.AI)",
    "Machine Learning (cs.LG)",
    "Machine Learning (stat.ML)"
  ],
  "doi": "",
  "journalName": "",
  "sourceType": "url"
}

**Comment by chtmp223** [comment]
{
  "_data": {
    "sourceId": "arxiv",
    "paperId": "2604.08423",
    "url": "https://arxiv.org/abs/2604.08423",
    "title": "Synthetic Data for any Differentiable Target",
    "authors": "Tristan Thrush, Sung Min Park, Herman Brunborg, Luke Bailey, Marcel Roed, Neil Band, Christopher Potts, Tatsunori Hashimoto",
    "abstract": "What are the limits of controlling language models via synthetic training data? We develop a reinforcement learning (RL) primitive, the Dataset Policy Gradient (DPG), which can precisely optimize synthetic data generators to produce a dataset of targeted examples. When used for supervised fine-tuning (SFT) of a target model, these examples cause the target model to do well on a differentiable metric of our choice. Our approach achieves this by taking exact data attribution via higher-order gradients and using those scores as policy gradient rewards. We prove that this procedure closely approximates the true, intractable gradient for the synthetic data generator. To illustrate the potential of DPG, we show that, using only SFT on generated examples, we can cause the target model's LM head weights to (1) embed a QR code, (2) embed the pattern 𝟼𝟽\\texttt{67}, and (3) have lower ℓ2\\ell^2 norm. We additionally show that we can cause the generator to (4) rephrase inputs in a new language and (5) produce a specific UUID, even though neither of these objectives is conveyed in the generator's input prompts. These findings suggest that DPG is a powerful and flexible technique for shaping model properties using only synthetic training examples.",
    "timestamp": "2026-04-10T18:06:15.607Z",
    "rating": "novote",
    "publishedDate": "2026/04/09",
    "tags": [
      "Computation and Language (cs.CL)",
      "Artificial Intelligence (cs.AI)",
      "Machine Learning (cs.LG)",
      "Machine Learning (stat.ML)"
    ],
    "doi": "",
    "journalName": "",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2026-04-10T18:06:16.147Z",
    "update_mode": "append",
    "issue_number": 296
  },
  "type": "initial_state"
}
